package.json

```diff
@@ -33,28 +33,29 @@
   "dependencies": {
     "@babel/cli": "^7.6.0",
     "@babel/core": "^7.6.0",
-    "babel-eslint": "^10.0.3",
-    "babel-loader": "^8.0.6",
     "@babel/helper-plugin-utils": "^7.0.0",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
     "@babel/preset-env": "^7.6.0",
     "@babel/preset-react": "^7.0.0",
-    "css-loader": "^3.2.0",
-    "copy-webpack-plugin": "^5.0.4",
+    "babel-eslint": "^10.0.3",
+    "babel-loader": "^8.0.6",
+    "bootstrap": "^4.3.1",
```

Comment:
Just an inquiry; we finally settled for bootstrap?
Thanks for re-ordering the packages in alphabetical order